lib/messager.rb

```diff
@@ -0,0 +1,23 @@
+require 'twilio-ruby'
+require 'date'
+require 'dotenv'
+
+class Messager
+  def initialize
+    @env = Dotenv.load
+  end
+
+  def send_text
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for send_text is too high. [16.03/15] (http://c2.com/cgi/wiki?AbcMetric)